### PR TITLE
i18n(zh-cn): Translate sidebar.mdx

### PR DESCRIPTION
**Pull request by liruifengv**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

- Translate sidebar.mdx

### CODE REVIEW

**Review by mingXta — APPROVED** [review]
lgtm!

File: docs/src/content/docs/zh/guides/sidebar.mdx [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+上面的配置生成以下侧边栏：
+
+<SidebarPreview
+	config={[
+		{ label: 'CSS & Styling', link: '/zh/guides/css-and-tailwind/' },
+		{ label: 'Astro', link: 'https://astro.build/' },
+	]}
+/>
+
+### 组合
```
Comment (Genteure) [inline docs/src/content/docs/zh/guides/sidebar.mdx:84]:
```suggestion
### 分组
```

`Group` 翻译成“分组”是不是更合理一点？

Comment (Genteure) [reply docs/src/content/docs/zh/guides/sidebar.mdx:84]: @liruifengv pinging for visibility

Comment (liruifengv) [reply docs/src/content/docs/zh/guides/sidebar.mdx]: Done.

**Review by Genteure — APPROVED** [review]
LGTM

**Review by delucis — APPROVED** [review]
Thank you @liruifengv! And thank you to @mingXta and @Genteure for reviewing 💖

**Pull request merged**